src/current/v22.2/install-cockroachdb-linux.md

```diff
@@ -164,8 +164,9 @@ true
 
 CockroachDB runtimes built for the ARM architecture have the following limitations:
 
-- CockroachDB on ARM is in [Limited Access](https://www.cockroachlabs.com/docs/stable/cockroachdb-feature-availability#feature-availability-phases) in CockroachDB v22.2.13, and is **experimental** in all other versions. Experimental binaries and Docker images are not qualified for production use and not eligible for support or uptime SLA commitments.
```

Comment:
A similar statement appears near the top of this page. Does that need to change as well?

Reply:
Good catch! Yes. I also will make a tweak to the 23.2 version of the file to align.